statsmodels/stats/oaxaca.py

```diff
@@ -398,33 +495,59 @@ class OaxacaResults:
     ----------
     params
         A list of all values for the fitted models.
+    std
+        A list of standard error calculations.
     """
-    def __init__(self, results, model_type, var_val=None):
+    def __init__(self, results, model_type, std_val=None):
         self.params = results
-        self.var = var_val
+        self.std = std_val
         self.model_type = model_type
 
     def summary(self):
         """
         Print a summary table with the Oaxaca-Blinder effects
         """
         if self.model_type == 2:
-            print(dedent("""\
-            Oaxaca-Blinder Two-fold Effects
-
-            Unexplained Effect: {:.5f}
-            Explained Effect: {:.5f}
-            Gap: {:.5f}""".format(
-                                self.params[0], self.params[1],
-                                self.params[2])))
-
+            if self.std is None:
```

Comment:
Ideally summary would return a Summary object which supports export to text and latex.  

Can be a different PR.

Reply:
I can try to do that. Now that there are more columns, I can use the summary table tool.

Reply:
@bashtage Do you have any references so that I can look into how to do what you asked? I am trying to add non-linear models then I will look into summary models. I have started working on diff-in-diff as well, so these might take a backseat, as with this PR, this module is extremely usable.